frontend/public/components/utils/dropdown.jsx

```diff
@@ -317,7 +317,6 @@ export class Dropdown extends DropdownMixin {
 
     _.each(items, (v, k) => addItem(k, v));
 
-    //Adding `dropDownClassName` specifically to use patternfly's context selector component, which expects `bootstrap-select` class on the dropdown. We can remove this additional property if that changes in upcoming patternfly versions.
```

Comment:
I assume we're now using `dropDownClassName` elsewhere?

Reply:
Yep.  To make dropdowns full width.